src/Dialect/ONNX/ElementsAttr/Strides.cpp

```diff
@@ -11,19 +11,19 @@
 #include "src/Dialect/ONNX/ElementsAttr/Strides.hpp"
 
 #include "src/Dialect/ONNX/ElementsAttr/Arrays.hpp"
+#include "src/Dialect/ONNX/ElementsAttr/StridesRange.hpp"
 
 using namespace mlir;
 
 namespace onnx_mlir {
 
-size_t getStridesPosition(
-    ArrayRef<int64_t> indices, ArrayRef<int64_t> strides) {
+size_t getStridesPosition(ArrayRef<uint64_t> index, ArrayRef<int64_t> strides) {
```

Comment:
Why index singular over indices?

Reply:
I use conventions from the mlir::ElementsAttr attribute interface methods.

I added a comment to make that clear.